example/src/screens/CreateRoom.tsx

```diff
@@ -90,7 +117,26 @@ export const CreateRoom = ({ navigation, route }: Props) => {
   }, []);
 
   return (
-    <BackgroundWrapper hasHeader>
+    <Animated.View
```

Comment:
I think it's a bit repetitive. Can we extract this to a separate component and reuse?

Reply:
Added a new component.